### ISSUE TITLE
Self-Built Rainmeter plugins don't work, packaged ones do

### ISSUE THREAD

**Issue by ScriptPup** [rainmeter/rainmeter#237]
I'm trying to develop a new plugin using the rainmeter documentation here https://docs.rainmeter.net/manual/plugins/.

I ran into an issue where it seemed like my plugin wasn't working, and decided to build the example plugins as a test to see whether I could make minor tweaks to get a better understanding of what was doing what, since I guessed I was missing something.

However, now it's looking like it's something wrong with the build, vs. anything I'm doing with my code. I've found essentially no information on how to build plugins, so I just assumed default visual studio settings and a right-click clean, right-click build on the project would work fine.

When I build the example "SystemVersion.dll" the dll is generated with no errors in the obj/x64/Debug file as expected. When I copy this .dll into my rainmeter/plugins folder, the log file shows that the plugin loads successfully but the plugin just shows "0" where all of the variable replacements should be happening (default skin value).

When I copy the pre-built "SystemVersion.dll" in the x64 folder into the rainmeter/plugins folder and launch the same skin, it works perfectly.

The pre-built dll is 1kb larger than the one I'm building on my system. Is there some kind of trick to building rainmeter dlls?

Visual Studio Version: 2019 community (Version 16.6.3)
OS: Windows 10
Rainmeter Version installed: 4.3.1 (64-bit)
API Version: (I didn't see any releases, but the commit ID was a32273c)

**Comment by brianferguson** [comment]
> When I copy the pre-built "SystemVersion.dll" in the x64 folder into the rainmeter/plugins folder and launch the same skin, it works perfectly.

I am not sure what you mean by "pre-built" ?

---

Everything seems to work for me:

![SystemVersion](https://user-images.githubusercontent.com/1591640/87271416-66f6b380-c490-11ea-9075-5512f0f7aa02.png)

I just opened the [SDK-CPP.sln](https://github.com/rainmeter/rainmeter-plugin-sdk/blob/master/C%2B%2B/SDK-CPP.sln) and build the SystemVersion project. I then copied the SystemVersion.dll (119KB) file to `%APPDATA%\Rainmeter\Plugins\` folder.

As you can see, I just used the simple test skin copied from the [PluginSystemVersion.cpp](https://github.com/rainmeter/rainmeter-plugin-sdk/blob/master/C%2B%2B/PluginSystemVersion/PluginSystemVersion.cpp#L15-L60) file.

I have no idea why it wouldn't work for you. 😕 

-Brian

**Comment by ScriptPup** [comment]
I'm using the C# version, so that's possibly one difference... Here's how it looks when I use the version I built.
![image](https://user-images.githubusercontent.com/6163646/87304087-435c5900-c4e2-11ea-83e4-794d0ec59e2b.png)

I meant the examples dlls that come with the project, but I just checked the .zip file and it's not included... So I guess my build must (for some reason) go to three different folders, and one of them works while the others don't.... That's very odd... But I think that at least solves it... Just needed to go back to step 0 and check my assumption.

So I guess I'll close it, since one of the dlls works.

Thanks!
